spec/controllers/hyrax/dashboard/collections_controller_spec.rb

```diff
@@ -74,7 +74,7 @@
 
       it "adds docs to the collection and adds the collection id to the documents in the collection" do
         post :create, params: {
-          batch_document_ids: [asset1.id],
+          batch_document_ids: [asset1.id, unowned_asset.id],
```

Comment:
Why are you doing this change?  It seems odd that no assertions are changing.

Reply:
This is to verify that only works the user has access to are added to the collection. This passes an ID of an unowned work with the request but that shouldn't change the original expectation that only `asset1` is added to the collection.